### ISSUE TITLE
TorrentToMedia.py post process issue Warning  Main  Found  no media files in 

### ISSUE THREAD

**Issue by shredthegnar** [clinton-hall/nzbToMedia#753]
Torrents are paused by the torrenttomedia script but then postprocessing appears to stop.   

Transmission download directory :   /media/Downloads/transmission/complete
Transmission incomplete directory:  /media/Downloads/transmission/incomplete
Inside  /media/Downloads/trasmission/complete are the folders  tv and movie

Looks like it finds the files, flattens the directory then cant find the files again.  When I navigate to /media/downloads/transmission/complete/tv the folder for the show is there and inside are the unpacked media files.

Attached a section of the logs with a file sent from couchpotato and sickrage

2015-04-28 19:49:07 INFO    ::MAIN: Loading config from [/usr/local/share/nzbToMedia/autoProcessMedia.cfg]
2015-04-28 19:49:07 INFO    ::MAIN: Checking database structure...
2015-04-28 19:49:07 DEBUG   ::MAIN: Checking Initial Schema database upgrade
2015-04-28 19:49:07 DEBUG   ::MAIN: InitialSchema upgrade not required
2015-04-28 19:49:07 DEBUG   ::MAIN: Checking if we can use git commands: git version
2015-04-28 19:49:07 DEBUG   ::MAIN: Executing git version with your shell in /usr/local/share/nzbToMedia
2015-04-28 19:49:07 DEBUG   ::MAIN: git version : returned successful
2015-04-28 19:49:07 DEBUG   ::MAIN: Using: git
2015-04-28 19:49:07 DEBUG   ::MAIN: Executing git symbolic-ref -q HEAD with your shell in /usr/local/share/nzbToMedia
2015-04-28 19:49:08 DEBUG   ::MAIN: git symbolic-ref -q HEAD : returned successful
2015-04-28 19:49:08 INFO    ::MAIN: Checking if git needs an update
2015-04-28 19:49:08 DEBUG   ::MAIN: Executing git rev-parse HEAD with your shell in /usr/local/share/nzbToMedia
2015-04-28 19:49:08 DEBUG   ::MAIN: git rev-parse HEAD : returned successful
2015-04-28 19:49:08 DEBUG   ::MAIN: Executing git fetch origin with your shell in /usr/local/share/nzbToMedia
2015-04-28 19:49:08 DEBUG   ::MAIN: git fetch origin : returned successful
2015-04-28 19:49:08 DEBUG   ::MAIN: Executing git rev-parse --verify --quiet "@{upstream}" with your shell in /usr/local/share/nzbToMedia
2015-04-28 19:49:08 DEBUG   ::MAIN: git rev-parse --verify --quiet "@{upstream}" : returned successful
2015-04-28 19:49:08 DEBUG   ::MAIN: Executing git rev-list --left-right "@{upstream}"...HEAD with your shell in /usr/local/share/nzbToMedia
2015-04-28 19:49:08 DEBUG   ::MAIN: git rev-list --left-right "@{upstream}"...HEAD : returned successful
2015-04-28 19:49:08 DEBUG   ::MAIN: cur_commit = 77da142216ba6dc91bb654635d4bd03f13c881e5 % (newest_commit)= 77da142216ba6dc91bb654635d4bd03f13c881e5, num_commits_behind = 0, num_commits_ahead = 0
2015-04-28 19:49:08 INFO    ::MAIN: No update needed
2015-04-28 19:49:08 INFO    ::MAIN: nzbToMedia Version:77da142216ba6dc91bb654635d4bd03f13c881e5 Branch:master (FreeBSD 9.1-RELEASE)
2015-04-28 19:49:08 WARNING ::MAIN: Failed to locate 7zip. Transcosing of disk images and extraction of .7z files will not be possible!
2015-04-28 19:49:08 DEBUG   ::MAIN: Connecting to transmission: http://localhost:9091
2015-04-28 19:49:08 INFO    ::MAIN: #########################################################
2015-04-28 19:49:08 INFO    ::MAIN: ## ..::[TorrentToMedia.py]::.. ##
2015-04-28 19:49:08 INFO    ::MAIN: #########################################################
2015-04-28 19:49:08 DEBUG   ::MAIN: Options passed into TorrentToMedia: ['/usr/local/share/nzbToMedia/TorrentToMedia.py']
2015-04-28 19:49:08 DEBUG   ::MAIN: Adding TORRENT download info for directory /media/Downloads/transmission/complete/movie to database
2015-04-28 19:49:08 DEBUG   ::MAIN: Received Directory: /media/Downloads/transmission/complete/movie | Name: The.Grey.2011.1080p.BluRay.x264-SPARKS | Category: 
2015-04-28 19:49:08 DEBUG   ::MAIN: SEARCH: Found Category: movie in directory structure
2015-04-28 19:49:08 INFO    ::MAIN: SEARCH: Found torrent directory The.Grey.2011.1080p.BluRay.x264-SPARKS in input directory directory /media/Downloads/transmission/complete/movie
2015-04-28 19:49:08 INFO    ::MAIN: SEARCH: Setting inputDirectory to /media/Downloads/transmission/complete/movie/The.Grey.2011.1080p.BluRay.x264-SPARKS
2015-04-28 19:49:08 DEBUG   ::MAIN: Determined Directory: /media/Downloads/transmission/complete/movie/The.Grey.2011.1080p.BluRay.x264-SPARKS | Name: The.Grey.2011.1080p.BluRay.x264-SPARKS | Category: movie
2015-04-28 19:49:08 INFO    ::MAIN: Auto-detected SECTION:CouchPotato
2015-04-28 19:49:08 DEBUG   ::MAIN: Stopping torrent The.Grey.2011.1080p.BluRay.x264-SPARKS in transmission while processing
2015-04-28 19:49:13 INFO    ::MAIN: Output directory set to: /media/downloads/transmission/complete/movie/The.Grey.2011.1080p.BluRay.x264-SPARKS
2015-04-28 19:49:13 DEBUG   ::MAIN: Scanning files in directory: /media/Downloads/transmission/complete/movie/The.Grey.2011.1080p.BluRay.x264-SPARKS
2015-04-28 19:49:13 DEBUG   ::MAIN: Found 2 files in /media/Downloads/transmission/complete/movie/The.Grey.2011.1080p.BluRay.x264-SPARKS
2015-04-28 19:49:13 DEBUG   ::MAIN: Checking for archives to extract in directory: /media/downloads/transmission/complete/movie/The.Grey.2011.1080p.BluRay.x264-SPARKS
2015-04-28 19:49:13 INFO    ::MAIN: FLATTEN: Flattening directory: /media/downloads/transmission/complete/movie/The.Grey.2011.1080p.BluRay.x264-SPARKS
2015-04-28 19:49:13 WARNING ::MAIN: Found no media files in /media/downloads/transmission/complete/movie/The.Grey.2011.1080p.BluRay.x264-SPARKS
2015-04-28 19:49:13 INFO    ::MAIN: Calling CouchPotato:movie to post-process:The.Grey.2011.1080p.BluRay.x264-SPARKS
2015-04-28 19:49:13 INFO    ::MAIN: Attemping imdbID lookup for The.Grey.2011.1080p.BluRay.x264-SPARKS
2015-04-28 19:49:13 INFO    ::MAIN: Searching folder and file names for imdbID ...
2015-04-28 19:49:13 INFO    ::MAIN: Searching IMDB for imdbID ...
2015-04-28 19:49:13 DEBUG   ::MAIN: Opening URL: http://www.omdbapi.com
2015-04-28 19:49:13 WARNING ::MAIN: Unable to find a imdbID for The.Grey.2011.1080p.BluRay.x264-SPARKS
2015-04-28 19:49:13 DEBUG   ::MAIN: Opening URL: http://localhost:5050/couchpotato/api/8c92504cd34644f4ac2eedbaf0e4462f/media.list with PARAMS: {}
2015-04-28 19:49:14 WARNING ::COUCHPOTATO: No media files found in directory /media/downloads/transmission/complete/movie/The.Grey.2011.1080p.BluRay.x264-SPARKS. Processing this as a failed download
2015-04-28 19:49:14 POSTPROCESS::COUCHPOTATO: FAILED DOWNLOAD DETECTED FOR The.Grey.2011.1080p.BluRay.x264-SPARKS
2015-04-28 19:49:14 POSTPROCESS::COUCHPOTATO: Setting failed release The.Grey.2011.1080p.BluRay.x264-SPARKS to ignored ...
2015-04-28 19:49:14 DEBUG   ::COUCHPOTATO: Opening URL: http://localhost:5050/couchpotato/api/8c92504cd34644f4ac2eedbaf0e4462f/release.ignore with PARAMS: {'id': u'77054205e09d428594a6b74954636274'}
2015-04-28 19:49:14 POSTPROCESS::COUCHPOTATO: SUCCESS: The.Grey.2011.1080p.BluRay.x264-SPARKS has been set to ignored ...
2015-04-28 19:49:14 POSTPROCESS::COUCHPOTATO: Trying to snatch the next highest ranked release.
2015-04-28 19:49:14 DEBUG   ::COUCHPOTATO: Opening URL: http://localhost:5050/couchpotato/api/8c92504cd34644f4ac2eedbaf0e4462f/movie.searcher.try_next
2015-04-28 19:49:44 POSTPROCESS::COUCHPOTATO: SUCCESS: Snatched the next highest release ...
2015-04-28 19:49:44 DEBUG   ::MAIN: Deleting torrent The.Grey.2011.1080p.BluRay.x264-SPARKS from transmission
2015-04-28 19:49:49 INFO    ::CLEANDIR: Directory /media/downloads/transmission/complete/movie/The.Grey.2011.1080p.BluRay.x264-SPARKS has been processed and removed ...
2015-04-28 19:49:49 INFO    ::MAIN: The /usr/local/share/nzbToMedia/TorrentToMedia.py script completed successfully.
2015-04-28 21:17:13 INFO    ::MAIN: Loading config from [/usr/local/share/nzbToMedia/autoProcessMedia.cfg]
2015-04-28 21:17:13 INFO    ::MAIN: Checking database structure...
2015-04-28 21:17:13 DEBUG   ::MAIN: Checking Initial Schema database upgrade
2015-04-28 21:17:13 DEBUG   ::MAIN: InitialSchema upgrade not required
2015-04-28 21:17:13 DEBUG   ::MAIN: Checking if we can use git commands: git version
2015-04-28 21:17:13 DEBUG   ::MAIN: Executing git version with your shell in /usr/local/share/nzbToMedia
2015-04-28 21:17:13 DEBUG   ::MAIN: git version : returned successful
2015-04-28 21:17:13 DEBUG   ::MAIN: Using: git
2015-04-28 21:17:13 DEBUG   ::MAIN: Executing git symbolic-ref -q HEAD with your shell in /usr/local/share/nzbToMedia
2015-04-28 21:17:13 DEBUG   ::MAIN: git symbolic-ref -q HEAD : returned successful
2015-04-28 21:17:13 INFO    ::MAIN: Checking if git needs an update
2015-04-28 21:17:13 DEBUG   ::MAIN: Executing git rev-parse HEAD with your shell in /usr/local/share/nzbToMedia
2015-04-28 21:17:13 DEBUG   ::MAIN: git rev-parse HEAD : returned successful
2015-04-28 21:17:13 DEBUG   ::MAIN: Executing git fetch origin with your shell in /usr/local/share/nzbToMedia
2015-04-28 21:17:13 DEBUG   ::MAIN: git fetch origin : returned successful
2015-04-28 21:17:13 DEBUG   ::MAIN: Executing git rev-parse --verify --quiet "@{upstream}" with your shell in /usr/local/share/nzbToMedia
2015-04-28 21:17:13 DEBUG   ::MAIN: git rev-parse --verify --quiet "@{upstream}" : returned successful
2015-04-28 21:17:13 DEBUG   ::MAIN: Executing git rev-list --left-right "@{upstream}"...HEAD with your shell in /usr/local/share/nzbToMedia
2015-04-28 21:17:13 DEBUG   ::MAIN: git rev-list --left-right "@{upstream}"...HEAD : returned successful
2015-04-28 21:17:13 DEBUG   ::MAIN: cur_commit = 77da142216ba6dc91bb654635d4bd03f13c881e5 % (newest_commit)= 77da142216ba6dc91bb654635d4bd03f13c881e5, num_commits_behind = 0, num_commits_ahead = 0
2015-04-28 21:17:13 INFO    ::MAIN: No update needed
2015-04-28 21:17:13 INFO    ::MAIN: nzbToMedia Version:77da142216ba6dc91bb654635d4bd03f13c881e5 Branch:master (FreeBSD 9.1-RELEASE)
2015-04-28 21:17:13 WARNING ::MAIN: Failed to locate 7zip. Transcosing of disk images and extraction of .7z files will not be possible!
2015-04-28 21:17:13 DEBUG   ::MAIN: Connecting to transmission: http://localhost:9091
2015-04-28 21:17:13 INFO    ::MAIN: #########################################################
2015-04-28 21:17:13 INFO    ::MAIN: ## ..::[TorrentToMedia.py]::.. ##
2015-04-28 21:17:13 INFO    ::MAIN: #########################################################
2015-04-28 21:17:13 DEBUG   ::MAIN: Options passed into TorrentToMedia: ['/usr/local/share/nzbToMedia/TorrentToMedia.py']
2015-04-28 21:17:13 DEBUG   ::MAIN: Adding TORRENT download info for directory /media/Downloads/transmission/complete/tv to database
2015-04-28 21:17:13 DEBUG   ::MAIN: Received Directory: /media/Downloads/transmission/complete/tv | Name: The.Flash.2014.S01E20.720p.HDTV.X264-DIMENSION[rarbg] | Category: 
2015-04-28 21:17:13 DEBUG   ::MAIN: SEARCH: Found Category: tv in directory structure
2015-04-28 21:17:13 INFO    ::MAIN: SEARCH: Found torrent directory The.Flash.2014.S01E20.720p.HDTV.X264-DIMENSION[rarbg] in input directory directory /media/Downloads/transmission/complete/tv
2015-04-28 21:17:13 INFO    ::MAIN: SEARCH: Setting inputDirectory to /media/Downloads/transmission/complete/tv/The.Flash.2014.S01E20.720p.HDTV.X264-DIMENSION[rarbg]
2015-04-28 21:17:13 DEBUG   ::MAIN: Determined Directory: /media/Downloads/transmission/complete/tv/The.Flash.2014.S01E20.720p.HDTV.X264-DIMENSION[rarbg] | Name: The.Flash.2014.S01E20.720p.HDTV.X264-DIMENSION[rarbg] | Category: tv
2015-04-28 21:17:13 INFO    ::MAIN: Auto-detected SECTION:SickBeard
2015-04-28 21:17:13 DEBUG   ::MAIN: Stopping torrent The.Flash.2014.S01E20.720p.HDTV.X264-DIMENSION[rarbg] in transmission while processing
2015-04-28 21:17:18 INFO    ::MAIN: Output directory set to: /media/downloads/transmission/complete/tv/The.Flash.2014.S01E20.720p.HDTV.X264-DIMENSION[rarbg]
2015-04-28 21:17:18 DEBUG   ::MAIN: Scanning files in directory: /media/Downloads/transmission/complete/tv/The.Flash.2014.S01E20.720p.HDTV.X264-DIMENSION[rarbg]
2015-04-28 21:17:18 DEBUG   ::MAIN: Found 2 files in /media/Downloads/transmission/complete/tv/The.Flash.2014.S01E20.720p.HDTV.X264-DIMENSION[rarbg]
2015-04-28 21:17:18 DEBUG   ::MAIN: Checking for archives to extract in directory: /media/downloads/transmission/complete/tv/The.Flash.2014.S01E20.720p.HDTV.X264-DIMENSION[rarbg]
2015-04-28 21:17:18 INFO    ::MAIN: FLATTEN: cd
2015-04-28 21:17:18 WARNING ::MAIN: Found no media files in /media/downloads/transmission/complete/tv/The.Flash.2014.S01E20.720p.HDTV.X264-DIMENSION[rarbg]
2015-04-28 21:17:18 INFO    ::MAIN: Calling SickBeard:tv to post-process:The.Flash.2014.S01E20.720p.HDTV.X264-DIMENSION[rarbg]
2015-04-28 21:17:18 INFO    ::MAIN: Attempting to auto-detect tv fork
2015-04-28 21:17:18 INFO    ::MAIN: SickBeard:tv fork auto-detection successful ...
2015-04-28 21:17:18 INFO    ::MAIN: SickBeard:tv fork set to sickrage
2015-04-28 21:49:43 INFO    ::MAIN: Loading config from [/usr/local/share/nzbToMedia/autoProcessMedia.cfg]
2015-04-28 21:49:43 INFO    ::MAIN: Checking database structure...
2015-04-28 21:49:43 DEBUG   ::MAIN: Checking Initial Schema database upgrade
2015-04-28 21:49:43 DEBUG   ::MAIN: InitialSchema upgrade not required
2015-04-28 21:49:43 DEBUG   ::MAIN: Checking if we can use git commands: git version
2015-04-28 21:49:43 DEBUG   ::MAIN: Executing git version with your shell in /usr/local/share/nzbToMedia
2015-04-28 21:49:43 DEBUG   ::MAIN: git version : returned successful
2015-04-28 21:49:43 DEBUG   ::MAIN: Using: git
2015-04-28 21:49:43 DEBUG   ::MAIN: Executing git symbolic-ref -q HEAD with your shell in /usr/local/share/nzbToMedia
2015-04-28 21:49:43 DEBUG   ::MAIN: git symbolic-ref -q HEAD : returned successful
2015-04-28 21:49:43 INFO    ::MAIN: Checking if git needs an update
2015-04-28 21:49:43 DEBUG   ::MAIN: Executing git rev-parse HEAD with your shell in /usr/local/share/nzbToMedia
2015-04-28 21:49:43 DEBUG   ::MAIN: git rev-parse HEAD : returned successful
2015-04-28 21:49:43 DEBUG   ::MAIN: Executing git fetch origin with your shell in /usr/local/share/nzbToMedia
2015-04-28 21:49:44 DEBUG   ::MAIN: git fetch origin : returned successful
2015-04-28 21:49:44 DEBUG   ::MAIN: Executing git rev-parse --verify --quiet "@{upstream}" with your shell in /usr/local/share/nzbToMedia
2015-04-28 21:49:44 DEBUG   ::MAIN: git rev-parse --verify --quiet "@{upstream}" : returned successful
2015-04-28 21:49:44 DEBUG   ::MAIN: Executing git rev-list --left-right "@{upstream}"...HEAD with your shell in /usr/local/share/nzbToMedia
2015-04-28 21:49:44 DEBUG   ::MAIN: git rev-list --left-right "@{upstream}"...HEAD : returned successful
2015-04-28 21:49:44 DEBUG   ::MAIN: cur_commit = 77da142216ba6dc91bb654635d4bd03f13c881e5 % (newest_commit)= 77da142216ba6dc91bb654635d4bd03f13c881e5, num_commits_behind = 0, num_commits_ahead = 0
2015-04-28 21:49:44 INFO    ::MAIN: No update needed
2015-04-28 21:49:44 INFO    ::MAIN: nzbToMedia Version:77da142216ba6dc91bb654635d4bd03f13c881e5 Branch:master (FreeBSD 9.1-RELEASE)
2015-04-28 21:49:44 WARNING ::MAIN: Failed to locate 7zip. Transcosing of disk images and extraction of .7z files will not be possible!

**Comment by shredthegnar** [comment]
I am sure someone with more experience would have determined the error much quicker than me but I fixed the issue I was having.  It was a case issue.  I set the output directory in the nzbtomediasettings as /media/downloads/transmission/tv  and it should have been /media/Downloads/transmission/tv. The lowercase d had the script looking in the wrong folder and thus giving me my error.   Hope this helps someone in the future.
